### PR TITLE
Make adjustment to QA testing issue template

### LINKED ISSUE #215: 🚧Make adjustment to QA testing issue template
### I have done the items below . . .

- [X] I have updated the title by replacing the '**_<title_**>' section.

### Description

Make various changes to improve the _**QA Testing Issue Template**_.  This improves clarity and understanding of the process.
The name of the file is `qa-testing-template.yml`

**Refer to the image below for more context:**
image

### Acceptance Criteria

**This issue is finished when:**
- [x] Use a label link to display the qa testing label for the checkbox below
image
- [x] Update the version syntax explanation in the info below the _**Version Of Velaptor Being Tested**_ section to the value below:
    - `v<major.minor.patch>[-preview-<prev-num>]`
- [x] Swap the _**Relevant Information**_ and _**Issues To Test**_ sections positionaly within the form.
- [x] Add period to the end of the sentence below
image
- [x] Issue template file added to the solution folder _**SolutionItems/GitHub/IssueTemplates**_
     - This is for ease of access within the IDEs **Visual Studio** and **JetBrains Rider**

### ToDo Items

- [x] Draft pull request created and linked to this issue
- [X] Priority label added to issue (**_low priority_**, **_medium priority_**, or **_high priority_**)
- [x] Issue linked to the proper project
- [X] Issue linked to proper milestone

### Issue Dependencies

_No response_

### Related Work

- KinsonDigital/issues-repo#9

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>

💡For more information on branching, refer to the projects [branching documentation](../../Documentation/Branching.md).

- [x] Pull request title matches the title of the associated issue.

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out and let's discuss it.

- [x] A preview-label label has been added to the pull request.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - This does not mean that the tests pass in both the local and CI environment.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

**Review by KinsonDigitalAdmin — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/qa-testing-template.yml [hunk 1]
```diff
@@ -20,47 +20,47 @@ body:
...
 
   - type: input
     id: version
     attributes:
       label: Version Of Velaptor Being Tested
       description: |
-        Enter the version of **Velaptor** that is being tested.  This is the version it 'will' become upon release.
-        Version Format: v<major.minor.patch>[-preview-<preview-num>]
+        Enter the version of **_Velaptor_** that is being tested.  This is the upcoming version that will be released.
+        Version Format: v<major.minor.patch>[-preview-<prev-num>]
```
Comment (kselena) [inline .github/ISSUE_TEMPLATE/qa-testing-template.yml:34]: Fix issues below:
```suggestion
        Enter the version of **_Velaptor_** that is being tested.  This is the version that **_WILL_** be released.
        Version Format: v<major.minor.patch>[-preview.<prev-num>]
```

Comment (kselena) [reply .github/ISSUE_TEMPLATE/qa-testing-template.yml:34]: Check other project issue templates for suggestions above.

File: .github/ISSUE_TEMPLATE/qa-testing-template.yml [hunk 2]
```diff
@@ -73,7 +73,7 @@ body:
           - #30 QA Testing Velaptor
       description: |
         A list of QA testing issues that are must be completed before this QA testing issue can be started.
```
Comment (kselena) [inline .github/ISSUE_TEMPLATE/qa-testing-template.yml:75]: Remove "are"
```suggestion
        A list of QA testing issues that must be completed before this QA testing issue can be started.
```

**Review by kselena — CHANGES_REQUESTED** [review]

**Review by kselena — APPROVED** [review]
